tasks/github-release.yaml

```diff
@@ -0,0 +1,82 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: build-go-binaries
```

Comment:
suggestion, keep the name in the same context as the file, for example: git-add-release-artifacts